dev-docs/coco/tracing.md

```diff
@@ -0,0 +1,48 @@
+# Tracing Kata
+
+Kata has first-class support for tracing via OpenTelemetry.
```

Comment:
We may want to link against the upstream document here: https://github.com/kata-containers/kata-containers/blob/main/docs/tracing.md